libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
```

Comment:
We should check only the field type here. I think field["name"] == "message" is coming from filebeat somehow, right?